### ISSUE TITLE
Adding an overload for EnableFeature ommitting the package name/path.

### ISSUE THREAD

**Issue by markrexwinkel** [jeffkl/ManagedDism#132]
I want to use this library to enable and disable features by using only the feature name. 
This is supported by the native API but the overloads are current not here. Currently you can only enable features using the feature name in combination with the package name or path.
Is it possible to add this overload ?

Something like:

```c#
public static void EnableFeature(DismSession session, string featureName, bool limitAccess = false, bool enableAll = false, List<string> sourcePaths = null, Microsoft.Dism.DismProgressCallback progressCallback = null, object userData = null)
{
       EnableFeature(session, featureName, null, DismPackageIdentifier.None, limitAccess, enableAll, sourcePaths, progressCallback, userData);
}
```

**Comment by jeffkl** [comment]
I'm a little confused by the documentation and assumed there was no way to only specify a feature name.

> FeatureName [in]
> The name of the feature that is being enabled. To enable more than one feature, separate each feature name with a semicolon.

> Identifier [in, optional]
> Optional. Either an absolute path to a .cab file or the package name for the parent package of the feature to be enabled.

> PackageIdentifier [in, optional]
> Optional. A valid DismPackageIdentifier enumeration value. DismPackageName should be used when the Identifier parameter is pointing to a package name, and DismPackagePath should be used when Identifier points to the absolute path of a .cab file. If Identifier field is not NULL, you must specify a valid PackageIdentifier parameter. If the Identifier field is NULL, the PackageIdentifier parameter is ignored.


But it looks like if you specify a `FeatureName`, `null` for `Identifier`, and anything for `PackageIdentifier` it could work?

Do you have a way to test adding this method by building the API locally?

```C#
/// <summary>
/// Enable a feature with the specified nae.
/// </summary>
/// <param name="session">A valid DISM Session. The DISM Session must be associated with an image. You can associate a session with an image by using the DismOpenSession Function.</param>
/// <param name="featureName">The name of the feature that is being enabled. To enable more than one feature, separate each feature name with a semicolon.</param>
/// <param name="limitAccess">Specifies whether Windows Update (WU) should be contacted as a source location for downloading files if none are found in other specified locations. Before checking WU, DISM will check for the files in the SourcePaths provided and in any locations specified in the registry by group policy. If the files required to enable the feature are still present on the computer, this flag is ignored.</param>
/// <param name="enableAll">Specifies whether to enable all dependencies of the feature. If the specified feature or any one of its dependencies cannot be enabled, none of them will be changed from their existing state.</param>
/// <param name="sourcePaths">A list of source locations to check for files needed to enable the feature.</param>
/// <param name="progressCallback">A progress callback method to invoke when progress is made.</param>
/// <param name="userData">Optional user data to pass to the DismProgressCallback method.</param>
/// <exception cref="DismException">When a failure occurs.</exception>
/// <exception cref="OperationCanceledException">When the user requested the operation be canceled.</exception>
/// <exception cref="DismRebootRequiredException">When the operation requires a reboot to complete.</exception>
public static void EnableFeatureByName(DismSession session, string featureName, bool limitAccess, bool enableAll, List<string> sourcePaths, Dism.DismProgressCallback progressCallback, object userData)
{
    EnableFeature(session, featureName, null, DismPackageIdentifier.None, limitAccess, enableAll, sourcePaths, progressCallback, userData);
}
```

If that works I'm more than happy to add it to the API.  I am having trouble testing it but it sounds like you might be able to?

**Comment by markrexwinkel** [comment]
I already tested it and it works. I am calling this method with featureName set to a semicolon separated list of feature names. 
Do you want me to create a pull request ? I have added the following overloads so it is consistent with the other overloads:

PublicAPI.Shipped.txt:
```c#
static Microsoft.Dism.DismApi.EnableFeature(Microsoft.Dism.DismSession session, string featureName, bool limitAccess, bool enableAll) -> void
static Microsoft.Dism.DismApi.EnableFeature(Microsoft.Dism.DismSession session, string featureName, bool limitAccess, bool enableAll, System.Collections.Generic.List<string> sourcePaths) -> void
static Microsoft.Dism.DismApi.EnableFeature(Microsoft.Dism.DismSession session, string featureName, bool limitAccess, bool enableAll, System.Collections.Generic.List<string> sourcePaths, Microsoft.Dism.DismProgressCallback progressCallback) -> void
static Microsoft.Dism.DismApi.EnableFeature(Microsoft.Dism.DismSession session, string featureName, bool limitAccess, bool enableAll, System.Collections.Generic.List<string> sourcePaths, Microsoft.Dism.DismProgressCallback progressCallback, object userData) -> void
```

**Comment by jeffkl** [comment]
Yes please, pull request gladly accepted!

**Comment by markrexwinkel** [comment]
I created pull request #134

**Comment by jeffkl** [comment]
Fixed in #134
